### DISCUSSION TITLE
How to use with integer user IDs?

### DISCUSSION THREAD

**Discussion by stumpylog** [LonelyVikingMichael/litestar-users#59] [Q&A] [answered]
From a line like https://github.com/LonelyVikingMichael/litestar-users/blob/8aab57403dc7a4725504bc03c9b81a2fa10c6308/litestar_users/user_handlers.py#L43 it appears this only supports using UUIDs as the user ID?  Is that correct?  Any way to update to allow integers (such as from `BigIntBase`)?

**Comment by LonelyVikingMichael** [comment, marked as answer]
Yes, this was an oversight on my part.
#56 aims to add identifier URI customization, which we can extend further to support `int` at the route and user handler level

**Comment by LonelyVikingMichael** [comment]
This should be resolved by #61
